portal-impl/src/com/liferay/portal/security/auth/AuthVerifierPipeline.java

```diff
@@ -66,8 +66,8 @@ public static String getAuthVerifierPropertyName(String className) {
 			PropsKeys.AUTH_VERIFIER, simpleClassName, StringPool.PERIOD);
 	}
 
-	public AuthVerifierPipeline(Map<String, Object> filterProperties) {
-		_filterProperties = filterProperties;
+	public AuthVerifierPipeline(Map<String, Object> initParametersMap) {
+		_initParametersMap = initParametersMap;
 
 		AuthVerifierTrackerCustomizer.addAuthVerifierPipeline(this);
```

Comment:
@arthurchan35 this can not be right. AuthVerifierPipeline is registered by AuthVerifierFilter which most like comes from modules. And modules come and go, but the way how you hold AuthVerifierPipeline is permanent. Even the module that contains the AuthVerifierFilter got undeploy, the AuthVerifierPipeline will stay in the map forever, which is leak.